backend/catalogue/models.py

```diff
@@ -150,7 +150,7 @@ class SaveState(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     hashstate = models.CharField(max_length = 8000)
     description = models.TextField(blank=True, null=True)
-    time_created = models.DateTimeField(default=datetime.now())
+    time_created = models.DateTimeField(default=django.utils.timezone.now)
```

Comment:
Hmm, I hadn't noticed that before; doesn't really matter here, but any reason that was explicitly a local-time?  (I'm used to defaulting to UTC in databases, unless there's a compelling reason otherwise)

Reply:
Yeah, this isn't really that important. I only changed it because the console was giving a warning for the field every time I started the server.